paasta_tools/cli/schemas/kubernetes_schema.json

```diff
@@ -505,6 +505,21 @@
                     },
                     "uniqueItems": true
                 },
+                "secret_volumes": {
+                    "type": "array",
+                    "items": {
+                        "type": "object",
+                        "properties": {
+                            "container_path": {
```

Comment:
The docs indicate this has to be a directory, so maybe container_dir would be better?

https://kubernetes.io/docs/concepts/configuration/secret/#using-secrets-as-files-from-a-pod

So the config would be something like:
```
  - container_dir: /home/nobody/.ssh/
    secret_name: id_rsa
```
And you'd have to make sure the secret name matches the filename you want it to be, I guess.

Reply:
Fixed by introducing `items`